src/lib/marketplaces/helper.ts

```diff
@@ -39,6 +39,50 @@ export function getTransfersFromInnerInstructions(
     });
 }
 
+function txContainsLog(tx: ParsedConfirmedTransaction, value: string): boolean {
+  if (!tx.meta?.logMessages) {
+    return false;
+  }
+  return Boolean(
+    tx.meta.logMessages.find((msg) => {
+      return msg.includes(value);
+    })
+  );
+}
+
+function getTokenOriginFromTx(
+  tx: ParsedConfirmedTransaction
+): string | undefined {
+  if (!tx.meta?.preTokenBalances?.length) {
+    return;
+  }
+
+  let balance = tx.meta?.preTokenBalances.find((balance) => {
+    if (!balance.uiTokenAmount.uiAmount) {
+      return false;
+    }
+    return balance.uiTokenAmount.uiAmount > 0;
+  });
+  if (balance) {
+    return (balance as TokenBalance & { owner?: string }).owner;
+  }
+
+  return;
+}
+
+function didTokenMovedInTx(tx: ParsedConfirmedTransaction): boolean {
```

Comment:
Removing the `d` in `Moved` makes this a little more readable, `didTokenMoveInTx` or maybe `wasTokenMovedInTx`.